packages/launcher/src/components/launch/Launch.tsx

```diff
@@ -69,16 +70,15 @@ export default function Launch(): ReactElement {
     const connected = await docker.isConnected();
     if (connected) {
       // setPulling(true);
-      setStatus("Pulling Axon image");
-      await docker.pullImage();
-      // setPulling(false);
-      setStatus("Finished pulling.");
-      // image downloaded
-      const containers = await docker.getContainers();
-      if (containers !== null && containers.length > 0) {
-        setStatus("Removing old containers");
-        await docker.reset();
-      }
+      dns.resolve("www.google.com", (err) => {
```

Comment:
Spoke about it on slack... instead of checking google, we should check if we can access the resources we need (docker)

Reply:
I am not sure this is the best way to check if we have internet. This only checks that we can resolve an address. If the user is using a local dns server, this check will pass when they really do not have a connection. This also has a timeout of 3 minutes which is a long time.

I recommend we instead send an HTTP request to `https://hub.docker.com/v2`. If the response is 200, then we can assume we have a good connection to docker hub.